plugins/reach/lib/model/VendorCatalogItem.php

```diff
@@ -278,6 +279,16 @@ public function getContract()
 		return $this->getFromCustomData(self::CUSTOM_DATA_CONTRACT);
 	}
 
+	public function setRequiresOverages($v)
+	{
+		$this->putInCustomData(self::CUSTOM_ADMIN_REQUIRES_OVERAGES, $v);
+	}
+
+	public function getRequiresOverages()
+	{
+		return $this->getFromCustomData(self::CUSTOM_ADMIN_REQUIRES_OVERAGES);
```

Comment:
set default value explicit to false (rather then default null) as it is boolean var

Reply:
done